include/ensmallen_bits/adam/adam.hpp

```diff
@@ -186,6 +187,8 @@ using NadaMax = AdamType<NadaMaxUpdate>;
 
 using OptimisticAdam = AdamType<OptimisticAdamUpdate>;
 
+using Padam = AdamType<PadamUpdate>;
```

Comment:
I guess that the user does not have any easy way through the constructor to set `partial`; do you think it is worth writing an extra constructor or something like this?